cermod/r1face_mic/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Copyright: (C) 2015 iCub Facility
```

Comment:
might be better to have a recent date ;-)

Reply:
Done, thanks